packages/scheduler/src/VirtualTimer.js

```diff
@@ -0,0 +1,60 @@
+/** @license MIT License (c) copyright 2010-2018 original author or authors */
```

Comment:
We've started switching to a [new copyright date format](https://github.com/mostjs/core/blob/master/packages/core/src/combinator/slice/bounds.js#L1).  Not all files use it yet, so it's not obvious--sorry about that--but, as we touch files, we're trying to remember to update it.  The new format only mentions the starting copyright date:

```js
/** @license MIT License (c) copyright 2010 original author or authors */
```